CMakeLists.txt

```diff
@@ -38,6 +38,11 @@ set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 if(RATS_TLS_BUILD_MODE STREQUAL "host")
     set(HOST 1)
     add_definitions(-DHOST)
+
+    include(FindSgxDcapQuoteVerify)
```

Comment:
According to verify_evidence.c in ecdsa, it depends on `sgx_urts.h sgx_dcap_quoteverify.h sgx_dcap_ql_wrapper.h `and so on. While these files are belongs to  `libsgx-headers package, libsgx-dcap-quote-verify-devel package, and libsgx-dcap-ql-devel package`. Should we check these packages as well?